README.md

```diff
@@ -1,14 +1,14 @@
-SherlockML incremental synchronization
+Faculty Platform incremental synchronization
 ======================================
```

Comment:
The underline length should match the title

Reply:
Thanks, will amend.